cpg_workflows/metamist.py

```diff
@@ -259,6 +266,49 @@ def __init__(self) -> None:
         self.default_dataset: str = get_config()['workflow']['dataset']
         self.aapi = AnalysisApi()
 
+    @retry(
+        stop=stop_after_attempt(3),
+        wait=wait_exponential(multiplier=3, min=8, max=30),
+        retry=retry_if_exception_type(ServiceException),
```

Comment:
is there a way for us to be alerted if too many of these exceptions are raised?

Reply:
We can revisit during Notification & Logging task, at the moment it won't be logged.